public/bustracker.html

```diff
@@ -1,282 +0,0 @@
-	id_to_png['HTC Desire C'] = 'black';
-	id_to_png['GT-I8190N'] = 'black';
-	id_to_png['blackberry'] = 'black';
-	id_to_png['HTC Desire S'] = 'red';
-	id_to_png['strawberry'] = 'red';
-
-	var id_to_clr = {};
-	id_to_clr['HTC Desire C'] = '#111';
-	id_to_clr['GT-I8190N'] = '#111';
-	id_to_clr['blackberry'] = '#111';
-	id_to_clr['HTC Desire S'] = '#D11';
-	id_to_clr['strawberry'] = '#D11';
-
-	// map borders
-    var maxLat = 51.7757;
-    var minLat = 51.755117;
-    var maxLng = -0.208826;
-    var minLng = -0.251999;
-    //var minLng = -0.235; // for testing
-    var normalBounds = new google.maps.LatLngBounds(new google.maps.LatLng(minLat, minLng), new google.maps.LatLng(maxLat, maxLng));
-    
-    var normalZone = new google.maps.Rectangle({
-        strokeColor: '#22DD22',
-        strokeOpacity: 0.7,
-        strokeWeight: 2,
-        fillColor: '#22DD22',
-        fillOpacity: 0.1,
-        bounds: normalBounds
-      });
-    
-	var llStation = new google.maps.LatLng(51.76373, -0.215564);
-	var llTitan = new google.maps.LatLng(51.762488, -0.243518);
-	var map;
-	var markers = new Array();
-	var lines = new Array();
-	var max_trail = 5;
-	
-	function initialize() {
-
-		var mapOptions = {
-			zoom : 14,
-			center : new google.maps.LatLng(51.764, -0.230)
-		};
-
-		map = new google.maps.Map(document.getElementById('map-canvas'),
-				mapOptions);
-
-		var marker = new google.maps.Marker({
-			position : llStation,
-			map : map,
-			title : 'Station'
-		});
-
-		var marker = new google.maps.Marker({
-			position : llTitan,
-			map : map,
-			title : 'Titan'
-		});
-
-        refreshJson();
-	    setInterval(refreshJson, 11500);
-	}
-	
-	function howLong(tsString) {
-        var d = Date.parse(tsString);
-        var now = Date.now();
-        var seconds = ((now - d) / 1000) >> 0;
-        return seconds;
-	}
-	
-	function formatinHMS(seconds) {
-        var s = (seconds % 60);
-        var m = ((seconds / 60) % 60) >> 0;
-        var h = (seconds / 3600) >> 0;
-        var hstr = String("0" + h).slice(-2);
-        var mstr = String("0" + m).slice(-2);
-        var sstr = String("0" + s).slice(-2);
-        return hstr + ':' + mstr + ':' + sstr;
-	}
-
-	function getAlphaString(n) {
-		var i = 1
-		if (n < i) {
-			return ''
-		} else if (n < i+1) {
-			return '75'
-		} else if (n < i+2) {
-			return '50'
-		} else {
-			return '25'
-		}
-	}
-	
-	function getRouteImage(device_id, n) {
-		var suffix = '.png'
-		var colour = id_to_png[device_id] || 'green';
-		var aa = getAlphaString(n)
-		var iconn = colour + aa + suffix
-		return iconn
-	}
-
-	function getOpacity(n) {
-		var i = 2
-		if (n < i) {
-			return 0.4
-		} else if (n < i+1) {
-			return 0.3
-		} else if (n < i+2) {
-			return 0.2
-		} else {
-			return 0.1
-		}
-	}
-	
-    function refreshJson() {
-	    $.getJSON('http://agile-journey-4782.herokuapp.com/gethistory/ocado?callback=?', function(jd) {
-	        var now = new Date();
-	        var myObject = eval(jd);
```

Comment:
The padded json has gone. If you are testing you can insert dummy data using cURL or the backdoor. So we don't need eval.